AutotypeURL/ATUAutotypeURL.m

```diff
@@ -51,14 +52,33 @@ - (nonnull NSString *)windowTitleForRunningApplication:(nonnull NSRunningApplica
 
 - (NSString *)_getURLFromSafari {
   NSAppleScript *script = [[NSAppleScript alloc] initWithSource:@"tell application \"Safari\" to get URL of front document"];
-  NSAppleEventDescriptor *aed = [script executeAndReturnError:NULL];
-  return aed.stringValue;
+    NSAppleEventDescriptor *aed = [script executeAndReturnError:NULL];
+    NSString *rawURL = aed.stringValue;
+  
+
+    NSLog(@"%@ - raw URL", rawURL);
+
+    NSLog(@"%@ - parsedURL", [self _parseURL:rawURL]);
+    return [self _parseURL:rawURL];
+    
+    
 }
 
 - (NSString *)_getURLFromChrome {
   NSAppleScript *script = [[NSAppleScript alloc] initWithSource:@"tell application \"Google Chrome\" to get URL of active tab of front window"];
   NSAppleEventDescriptor *aed = [script executeAndReturnError:NULL];
   return aed.stringValue;
+}
+//Additions for parsing the url subdomain.root.com
+- (NSString *)_parseURL: (NSString*) url;
```

Comment:
the opening bracket should be on the same line and the trailing semicolon is a funny thing to do :)

Reply:
haha...can you tell i am not a Obj-C coder?? I spent 5 minutes trying to figure out that syntax error which resulted in this semicolon.

but now it's fixed.